tensorrt_llm/_torch/modules/fused_moe/quantization.py

```
@@ -4466,6 +4479,158 @@ def setup_quant_scales(self, module: torch.nn.Module):
+        sf_pack_k = self.BLOCK_SIZE * self.BLOCK_SCALES_VEC_SIZE  # = 128
+        assert module.hidden_size % sf_pack_k == 0, (
+            f"hidden_size={module.hidden_size} must be a multiple of "
+            f"{sf_pack_k} for MXFP8 MoE int32 SF packing")
+        assert module.intermediate_size_per_partition % sf_pack_k == 0
+
+        w3_w1_weight_scale = nn.Parameter(torch.empty(
+            (module.expert_size_per_partition,
+             module.expand_intermediate_size_per_partition,
+             module.hidden_size // sf_pack_k),
+            dtype=self.BLOCK_SCALES_DTYPE),
+                                          requires_grad=False)
```

Flake8 (7.3.0)

[error] continuation line over-indented for hanging indent

(E126)